app-modules/prospect/src/Models/ProspectSource.php

```diff
@@ -38,10 +42,11 @@
  *
  * @mixin Eloquent
  */
-class ProspectSource extends BaseModel
+class ProspectSource extends BaseModel implements Auditable
 {
     use HasUuids;
     use SoftDeletes;
+    use AuditableTrait;
```

Comment:
We should probably add this trait to the `BaseModel` as well.

Reply:
I don't think we can at this point. Because the ADM Models extend the `BaseModel` and they don't have this trait